src/main/webapp/i18n/de/exam.json

```diff
@@ -247,6 +247,7 @@
                 "lastName": "Nachname",
                 "email": "E-Mail",
                 "registrationNumber": "Matrikelnummer",
+                "registerAllFromCourse": "Registriere alle zu diesem Kurs angemeldeten Studenten",
```

Comment:
This is too long for a button. I suggest to add a tooltip and make the button text shorter, e.g.

Reply:
```suggestion
                "registerAllFromCourse": "Registriere Kurs Studierende",
```

Reply:
done!

Reply:
done!